extensions/extended-blocks/paid-blocks/class-paid-blocks.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Block Editor class for Plan Blocks.
+ * Sets blocks as paid.
```

Comment:
> Sets blocks as paid.

I'm not sure this is fully accurate as to the responsibilities or behaviour of this class. 

It "marks a block's availability as needing to be checked against the current plan".

Reply:
Going to update the text